openpype/modules/kitsu/plugins/publish/integrate_kitsu_note.py

```diff
@@ -96,20 +96,12 @@ def get_settings_from_context(self, context):
             for setting_name, setting_value in settings_from_context.items():
                 self.log.info(f"- {setting_name}: {setting_value}")
 
-    def skip_instance(self, context, instance, kitsu_task: dict) -> bool:
+    def skip_instance(self, instance, kitsu_task: dict) -> bool:
         """Define if the instance needs to be skipped or not.
 
         Returns:
             bool: True if the instance needs to be skipped. Else False.
         """
-        # Check already existing comment
-        if context.data.get("kitsu_comment"):
-            self.log.info(
-                "Kitsu comment already set, "
-                "skipping comment creation for instance..."
-            )
-            return True
```

Comment:
Keep that

Reply:
Why ? That's the point of these changes: we remove the kitsu_comment from the context to let it to the instance.  
And we work on instances now.  
So this check means nothing anymore.

Also we want to be able to add multiple reviews for multiple families, except for the `Workfile - Only` which must not be triggered if any other family already has a note.

No ?

Reply:
I add it back but using only instance

Reply:
Modified in https://github.com/Tilix4/OpenPype/pull/80/commits/4597e704b576488f2d9d93b2d3abbab438a8a6b6